### ISSUE TITLE
Deadline quota

### ISSUE THREAD

**Issue by Sheep667** [HolographicWings/LethalExpansion#93]
Hello, it's when i reach the last day in my quota and i comeback from a custom moons the games skips the day to sell and goes directly to the you failed to reach the quota. From my test the day isn't skipped in solo but in multiplayer it does. I'm currently forced to use normal moons to finish my quota.

**Comment by HolographicWings** [comment]
> Hello, it's when i reach the last day in my quota and i comeback from a custom moons the games skips the day to sell and goes directly to the you failed to reach the quota. From my test the day isn't skipped in solo but in multiplayer it does. I'm currently forced to use normal moons to finish my quota.

Which moon ?

**Comment by Sheep667** [comment]
Any custom moon (i have like 6 of them), and i was hosting

**Comment by HolographicWings** [comment]
> Any custom moon (i have like 6 of them), and i was hosting

does it happen only with LE installed too ?

**Comment by Sheep667** [comment]
Only on moons that depends on LE, cause like i said normal moon and custom object doesn't make the bug.

**Comment by HolographicWings** [comment]
> Only on moons that depends on LE, cause like i said normal moon and custom object doesn't make the bug. If you want i can record a video and send you the logs output with it

yes but i asked you if it happen if you only have LE and theses moons of installed
not any other mod

**Comment by Sheep667** [comment]
Seem to work when only with lethal expansion, mb if i wasted your time, since it's only did on custom moon

**Comment by HolographicWings** [comment]
> Seem to work when only with lethal expansion, mb if i wasted your time, since it's only did on custom moon

So it's an incompatibility
Could you try to find which mod is incompatible? That would be helpful to fix the issue
